documentation/IDTA-01001/modules/ROOT/pages/changelog.adoc

```diff
@@ -93,6 +93,11 @@ This means for certification a maximum of 32 recursion test cases should be test
 * CHANGED: Transfer from .docx to asciidoc (.adoc) and maintenance in GitHub (including (https://github.com/admin-shell-io/aas-specs/issues/318[#318], https://github.com/admin-shell-io/aas-specs/issues/316[#316] )
 * CHANGED: Transfer of all UML figures to PlantUML (.puml), changed notation for model references; maintenance in GitHub, no XMI presenation part of release any longer (https://github.com/admin-shell-io/aas-specs/issues/439[#439]) 
 
+Bugfixes:
+
+* CHANGED: add missing `<referredSemanticId>` to grammar for textual representation of References https://github.com/admin-shell-io/aas-specs/issues/557[#557])
+* CHANGED: examples for LangStringSet (correct "DE" to "de" etc. and enhance examples)
```

Comment:
```suggestion
* CHANGED: enhance and correct examples for LangStringSet so that they correspond to their concrete serialization in XML, JSON and RDF
```